src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStartTask.java

```diff
@@ -41,13 +43,23 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
     this.cloudSdkBuilderFactory = cloudSdkBuilderFactory;
   }
 
+  public void setDevAppServerLoggingDir(File devAppServerLoggingDir) {
+    this.devAppServerLoggingDir = devAppServerLoggingDir;
+  }
+
+  @OutputDirectory
+  public File getDevAppServerLoggingDir() {
+    return devAppServerLoggingDir;
+  }
+
   /** Task entrypoint : start the dev appserver (non-blocking). */
   @TaskAction
   public void startAction() throws AppEngineException, IOException {
 
     // Add a listener to write to a file for non-blocking starts, this really only works
     // when the gradle daemon is running (which is default for newer versions of gradle)
-    File logFile = File.createTempFile("dev_appserver", ".out", getTemporaryDir());
+    File logFile =
+        new File(devAppServerLoggingDir, "dev_appserver" + System.currentTimeMillis() + ".out");
```

Comment:
So I think it's pretty common that gradle/maven users run clean somewhat often (habit, probably), but I was thinking about this over the weekend, specifically users who wish to have a post-run task in an IDE can trigger a log reader to launch if the log file name is consistent. 
The downside is that old logs will be erased, errors should be repeatable though (and gradle users can presumably trigger a preserve logs task on their own before running a new appserver).

So yeah, lets overwrite the file.